lib/unit-ruby/util/api_resource.rb

```diff
@@ -70,7 +70,9 @@ def self.resource_path(id)
       "#{path}/#{id}"
     end
 
-    def self.resources_path
+    def self.resources_path(id = nil)
```

Comment:
i think these two actions are the first `create`s that take an id, which is why I've made this change so that we can post to `/customers/:customerId/token/verification` etc